src/main/java/com/jcabi/aspects/Loggable.java

```diff
@@ -132,6 +132,12 @@
      */
     int value() default Loggable.INFO;
 
+    /**
+     * Log level for exceptions. If not defined then the log level of value is used.
+     * @return The log level
+     */
+    int exceptionLevel() default -1;
```

Comment:
should be a constant here, probably `Loggable.WARNING`

Reply:
Creating a constant on this class would cause it to be visible to developers using the annotation, which could cause confusion. At most I could create a class for the constant, so it would be less confusing to developers using the library.
Should I proceed with that implementation or is it overkill?

Reply:
@fnmps I don't quite get why we need a class. `-1` means default, so you can add `Loggable.DEFAULT`. `onException` probably would be a better name than `exceptionLevel`